tests/test_modern.py

```diff
@@ -20,10 +21,12 @@
 import pyclean.modern
 from pyclean.modern import (
     delete_filesystem_objects,
+    normalize_path,
```

Comment:
Rename here and all references below, including in the docstrings.

```suggestion
    normalize,
```